### PR TITLE
YLP-1444 Handle user name display for Japanese

### PR DESCRIPTION
**Pull request by celia-rouis**

### CODE REVIEW

**Comment by celia-rouis** [comment]
Related yourloops PR: https://github.com/mdblp/yourloops/pull/816

**Review by Nas84 — APPROVED** [review]
LGTM

File: packages/yourloops/components/patient/patient-row.tsx [hunk 1]
```diff
@@ -277,7 +279,7 @@ const PatientRow: FunctionComponent<PatientRowProps> = ({ patient, filter }) =>
           >
             <Box>
               <IconActionButton
-                ariaLabel={`${t('remove-patient')}-${patient.profile.email}`}
+                ariaLabel={`${t('remove-patient')}-${email}`}
```
Comment (TimCanu) [inline packages/yourloops/components/patient/patient-row.tsx:282]: opt: This aria label is not really correct. It should have a value similar to `Remove patient tim@diabeloop.fr`.
I write this comment as optional as it was already like that before.

**Review by TimCanu — APPROVED** [review]
Nice job!

**Pull request merged**